core/src/test/scala/scalacheck/arbitrary/package.scala

```diff
@@ -0,0 +1,19 @@
+package scalacheck
```

Comment:
I'd prefer to scope this inside `pureconfig`. Using `scalacheck` may mislead people into thinking this is a built-in package from scalacheck.